py/makemoduledefs.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+
+# This pre-processor parses provided objects' c files for 
+# `MP_REGISTER_MODULE(module_name, obj_module, enabled_define)`
+# These are used to generate a header with the required entries for 
+# `mp_rom_map_elem_t mp_builtin_module_table[]` in py/objmodule.c
+
+from __future__ import print_function
+
+import re
+import os
+import argparse
+
+
+def find_c_file(vpath, obj_file):
+    """
+    Search the provided vpaths for the c file that matches the provided object_file
```

Comment:
Shouldn't docstring style be something more standard like
```
""" Short description which ends with period.

Rest of description
"""

or

```
"""  Short description which ends with period. """
```

Just checking, don't know if this is set in stone somewhere for uPy.

Reply:
I've never loved the first line starting on the same line as the opening """, but it does seem to be the more standard convention in PEP 257.